### ISSUE TITLE
[SF][A11y][ui5-dialog] Unable to resize the resizable dialog using keyboard

### ISSUE THREAD

**Issue by 2bno1** [SAP/ui5-webcomponents#4666]
### **Bug Description**
Since this dialog is resizable via mouse, the same functionality must be available for keyboard users also.
In SAP UI5 dialogs, SHIFT+CNTL+arrow keys is the standard shortcut for resizing as explained in the recording above. Same must be implemented for this dialog also. To be consistent, WC dialogs should adopt the same standard keyboard shortcuts as SAP UI5 for resizing.

SAP UI5 dialog Reference: 
1. Goto https://sapui5.hana.ondemand.com/#/entity/sap.m.Dialog/sample/sap.m.sample.Dialog
2. Click 'Dialog (Resizable)'
3. Press SHIFT+CNTL+arrow keys, notice that the dialog is resizing

### **Context**
- UI5 Web Components version: 1.0.2
- OS/Platform: macOS
- Browser: Chrome
- Affected component: ui5-dialog

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

**Comment by Shtilianova** [comment]
Hello @SAP/ui5-webcomponents-topic-rd, take a look of this request.

**Comment by gmkv** [comment]
Hi @2bno1 

The dialog does support dragging and resizing with the keyboard.
This was already implemented with https://github.com/SAP/ui5-webcomponents/issues/2082.
[Here's a sample](https://codesandbox.io/s/ui5-webcomponents-forked-ofm0z1?file=/index.html). Note the usage of the `resizable` and `draggable` attributes.

**Comment by 2bno1** [comment]
Hi @gmkv 
Thanks for your reply.

I tried your [sample](https://codesandbox.io/s/ui5-webcomponents-forked-ofm0z1?file=/index.html), but I'm not able to resize the dialog with the keyboard (CTRL + SHIFT + arrow keys). 

I also debugged the demo and found the `_onDragOrResizeKeyDown` in `Dialog` is not called when I was pressing the keyboard after opening the dialog.

<img width="1464" alt="Screen Shot 2022-03-09 at 9 58 05 AM" src="https://user-images.githubusercontent.com/1751846/157358136-726b8c75-62ab-4fff-a902-5fa0f883899a.png">

**Comment by ilhan007** [comment]
Hello @2bno1 
To drag -> Arrows Up, ArrowDown, ArrowLeft and ArrowRight, 
To resize > SHIFT + ArrowUp,  SHIFT + ArrowDown,  SHIFT + ArrowLeft and SHIFT + ArrowRight


https://user-images.githubusercontent.com/15702139/157387816-5e9477af-5f28-4b3a-ab18-7441a1e1f5fa.mov

**Comment by gmkv** [comment]
@2bno1 
The resizing keyboard combination is defined as [Shift]+[Arrow keys] in the Fiori Keyboard Interaction spec. Where did you read that it should be `CTRL + SHIFT + arrow keys`?

**Comment by 2bno1** [comment]
@ilhan007 Thanks for the reply. `CTRL` + `arrow keys` works for me.

**Comment by 2bno1** [comment]
> @2bno1
> The resizing keyboard combination is defined as [Shift]+[Arrow keys] in the [Fiori Keyboard Interaction spec](https://wiki.wdf.sap.corp/wiki/display/Acc/Fiori+Keyboard+Interaction). Where did you read that it should be `CTRL + SHIFT + arrow keys`?

@gmkv In [UI5 sample dialog page](https://sapui5.hana.ondemand.com/#/entity/sap.m.Dialog/sample/sap.m.sample.Dialog), dialog can be resized by `CTRL` + `SHIFT` + `arrow keys` (but `SHIFT` + `arrow keys` also works)

In this [sample](https://codesandbox.io/s/ui5-webcomponents-forked-ofm0z1?file=/index.html), `SHIFT` + `arrow keys` can resize the dialog, however, `CTRL` + `SHIFT` + `arrow keys` does not resize the dialog.

**Comment by gmkv** [comment]
How OpenUI5 handles keyboard combination (ignoring whether CTRL is pressed or not, when pressing Shift + Arrow keys to resize the sap.m.Dialog) is not relevant to how the ui5-webcomponents dialog component handles resizing.

The keyboard handling combination is implemented as defined (Shift + Arrow keys) in the Fiori Keyboard Interaction spec.
